_includes/privacy_policy_export.html

```diff
@@ -0,0 +1,202 @@
+</td>
+</tr>
+<tr>
+<td>
+<p><strong>Website</strong></p>
+</td>
+<td>
+<p><span style="font-weight: 400;">the website that you are currently using, socratesuk.org, and any sub-domains of this site unless expressly excluded by their own terms and conditions.</span></p>
+</td>
+</tr>
+</tbody>
+</table>
+<p><span style="font-weight: 400;">2. In this privacy policy, unless the context requires a different interpretation:</span></p>
+<ol style="list-style-type: lower-alpha;">
+<li><span style="font-weight: 400;">the singular includes the plural and vice versa;</span></li>
+<li><span style="font-weight: 400;">references to sub-clauses, clauses, schedules or appendices are to sub-clauses, clauses, schedules or appendices of this privacy policy;</span></li>
+<li><span style="font-weight: 400;">a reference to a person includes firms, companies, government entities, trusts and partnerships;</span></li>
+<li><span style="font-weight: 400;">"including" is understood to mean "including without limitation";</span></li>
+<li><span style="font-weight: 400;">reference to any statutory provision includes any modification or amendment of it;</span></li>
+<li><span style="font-weight: 400;">the headings and sub-headings do not form part of this privacy policy.</span></li>
+</ol>
+<p><strong>Scope of this privacy policy</strong></p>
+<p><span style="font-weight: 400;">3. This privacy policy applies only to the actions of SOFTWARE CRAFT AND TESTING UK CIC t/a SoCraTes UK and Users with respect to this Website. It does not extend to any websites that can be accessed from this Website including, but not limited to, any links we may provide to social media websites.</span></p>
+<p><span style="font-weight: 400;">4. For purposes of the applicable Data Protection Laws, SOFTWARE CRAFT AND TESTING UK CIC t/a SoCraTes UK is the "data controller". This means that SOFTWARE CRAFT AND TESTING UK CIC t/a SoCraTes UK determines the purposes for which, and the manner in which, your Data is processed.</span></p>
+<p><strong>Data collected</strong></p>
+<p><span style="font-weight: 400;">5. We may collect the following Data, which includes personal Data, from you:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">name;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">contact Information such as email addresses and telephone numbers;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">demographic information such as postcode, preferences and interests;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">financial information such as credit / debit card numbers;</span></li>
```

Comment:
Do we ever collect financial information? It feels like all of that is handled elsewhere? (maybe this includes things likethings like invoice addresses?)

Reply:
We are more likely to now we have our own bank account. Currently evenbrite handles all the processing so it's more of a future proof statement.